src/components/cookies-banner/_cookies-banner.scss

```diff
@@ -8,3 +8,14 @@
   color: $cookie-banner-text;
   word-break: break-word;
 }
+
+.btn--secondary {
+  margin-left: 0.5rem;
+}
+
+.btn--cookies {
```

Comment:
I don't think we should solve this problem in this way. I would use utility classes on the element in the mark up instead of creating classes to handle layout in this way.

I think there are some classes to handle width and breakpoints i.e. `u-wa@xs@s` etc